### PR TITLE
[Docs] Fix how lfs data path is set

### PR DESCRIPTION
**Pull request by 6543**
inconsistency was introduced in #12813 & #12988

### CODE REVIEW

**Comment by noerw** [comment]
Shouldnt one of `[server] LFS_CONTENT_PATH` & `[lfs] PATH` be deprecated (+ undocumented) instead of having two keys for the same thing?

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 1]
```diff
@@ -882,7 +882,7 @@ is `data/lfs` and the default of `MINIO_BASE_PATH` is `lfs/`.
 
 - `STORAGE_TYPE`: **local**: Storage type for lfs, `local` for local disk or `minio` for s3 compatible object storage service or other name defined with `[storage.xxx]`
 - `SERVE_DIRECT`: **false**: Allows the storage driver to redirect to authenticated URLs to serve files directly. Currently, only Minio/S3 is supported via signed URLs, local does nothing.
-- `CONTENT_PATH`: **./data/lfs**: Where to store LFS files, only available when `STORAGE_TYPE` is `local`.
+- `PATH`: **./data/lfs**: Where to store LFS files, only available when `STORAGE_TYPE` is `local`. Default is LFS_CONTENT_PATH value in [server] section.
```
Comment (lunny) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]: This is compatible with old configuration. We have to add a depreciate here.

File: docs/content/doc/advanced/config-cheat-sheet.zh-cn.md [hunk 2]
```diff
@@ -75,6 +75,7 @@ menu:
 
 - `LFS_START_SERVER`: 是否启用 git-lfs 支持. 可以为 `true` 或 `false`， 默认是 `false`。
 - `LFS_JWT_SECRET`: LFS 认证密钥，改成自己的。
+- `LFS_CONTENT_PATH`: 存放 lfs 命令上传的文件的地方，默认是 `data/lfs`。
```
Comment (6543) [inline docs/content/doc/advanced/config-cheat-sheet.zh-cn.md]: @lunny how to write deprecated in cn?

Comment (lunny) [reply docs/content/doc/advanced/config-cheat-sheet.zh-cn.md]: You can use `已废弃` :)

Comment (6543) [inline docs/content/doc/advanced/config-cheat-sheet.zh-cn.md]:
```suggestion
- `LFS_CONTENT_PATH`: **已废弃**, 存放 lfs 命令上传的文件的地方，默认是 `data/lfs`。
```

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md (hunk 1 above)
Comment (6543) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]:
```suggestion
- `PATH`: **./data/lfs**: Where to store LFS files, only available when `STORAGE_TYPE` is `local`. Default but deprecated is LFS_CONTENT_PATH value in [server] section.
```

File: docs/content/doc/advanced/config-cheat-sheet.zh-cn.md [hunk 3]
```diff
@@ -326,7 +327,7 @@ LFS 的存储配置。 如果 `STORAGE_TYPE` 为空，则此配置将从 `[stora
 
 - `STORAGE_TYPE`: **local**: LFS 的存储类型，`local` 将存储到磁盘，`minio` 将存储到 s3 兼容的对象服务。
 - `SERVE_DIRECT`: **false**: 允许直接重定向到存储系统。当前，仅 Minio/S3 是支持的。
-- `CONTENT_PATH`: 存放 lfs 命令上传的文件的地方，默认是 `data/lfs`。
+- `PATH`: **[server] LFS_CONTENT_PATH**: 存放 lfs 命令上传的文件的地方，默认是 `data/lfs`。
```
Comment (6543) [inline docs/content/doc/advanced/config-cheat-sheet.zh-cn.md]:
```suggestion
- `PATH`: 存放 lfs 命令上传的文件的地方，默认是 `data/lfs`。
```

**Review by 6543 — COMMENTED** [review]
OK, think it's ready to look at again :)

**Review by noerw — APPROVED** [review]

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 4]
```diff
@@ -882,7 +882,7 @@ is `data/lfs` and the default of `MINIO_BASE_PATH` is `lfs/`.
 
 - `STORAGE_TYPE`: **local**: Storage type for lfs, `local` for local disk or `minio` for s3 compatible object storage service or other name defined with `[storage.xxx]`
 - `SERVE_DIRECT`: **false**: Allows the storage driver to redirect to authenticated URLs to serve files directly. Currently, only Minio/S3 is supported via signed URLs, local does nothing.
-- `CONTENT_PATH`: **./data/lfs**: Where to store LFS files, only available when `STORAGE_TYPE` is `local`.
+- `PATH`: **./data/lfs**: Where to store LFS files, only available when `STORAGE_TYPE` is `local`. Default but deprecated is LFS_CONTENT_PATH value in [server] section.
```
Comment (6543) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]:
```suggestion
- `PATH`: **./data/lfs**: Where to store LFS files, only available when `STORAGE_TYPE` is `local`. If not set it fall back to deprecated LFS_CONTENT_PATH value in [server] section.
```

**Review by lunny — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**